homework_04/18_stelian_todorichkov/test_circle.py

```diff
@@ -0,0 +1,53 @@
+"""
+File with tests
+"""
+from circle import Circle, Point, RealativePosition
+
+def test_same():
+    """
+    Test circles are same
+    """
+    circle1 = Circle(Point(0.0, 0.0), 5.0)
+    circle2 = Circle(Point(0.0, 0.0), 5.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.SAME
+
+def test_intersecting():
+    """
+    Test circles are intersecting
+    """
+    circle1 = Circle(Point(0.0, 0.0), 2.0)
+    circle2 = Circle(Point(5.0, 0.0), 5.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.INTERSECTING
+
+def test_touching():
+    """
+    Test circles are touching
+    """
+    #outside
+    circle1 = Circle(Point(-1.0, 1.0), 3.0)
+    circle2 = Circle(Point(3.0, -2.0), 2.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.TOUCHING
+
+    #inside
+    circle1 = Circle(Point(-1.0, 0.0), 3.0)
+    circle2 = Circle(Point(3.0, -3.0), 8.0)
+    assert circle1.find_relative_position(circle2) == RealativePosition.TOUCHING
+
+def test_no_common_points():
```

Comment:
I really like the comments on the fourth test - they help me understand your logic.